README.md

```diff
@@ -101,6 +120,9 @@ wildfly:
    title-check:
      pattern: "\\[WFLY-\\d+\\]\\s+.*|WFLY-\\d+\\s+.*"
      message: "Wrong content of the title!"
+ emails:
```

Comment:
we should probably call it only email, so not to expect multiple emails

Reply:
We have for example `regexes` under description and we allow only single regex. Isn't it the same?